meson_options.txt

```diff
@@ -63,11 +63,11 @@ option('key_screen', type: 'boolean',
   description: 'Enable the key editor screen')
 
 option('lyrics_screen', type: 'boolean',
-  value: false,
+  value: true,
   description: 'Enable the lyrics screen')
 
 option('lyrics_plugin_dir', type: 'string',
-  value: '',
+  value: '/usr/lib/ncmpc/lyrics/',
```

Comment:
This doesn't belong here either. And is a bad change, because it hard-codes a path. You'd better use something relative to the install prefix. But let's not use this PR for such things - let's get your new plugin merged first, and then talk about other suggestions.